checker/checker.go

```diff
@@ -235,9 +236,9 @@ func (c *Checker) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	errs := make([]*pb.ProcessError, 0, 1)
 	result, err := check.Do(cctx, c.checkList)
 	if err != nil {
-		errs = append(errs, unit.NewProcessError(pb.ErrorType_CheckFailed, err.Error()))
+		errs = append(errs, unit.NewProcessError(pb.ErrorType_CheckFailed, err))
 	} else if !result.Summary.Passed {
-		errs = append(errs, unit.NewProcessError(pb.ErrorType_CheckFailed, "check was failed, please see detail"))
+		errs = append(errs, unit.NewProcessError(pb.ErrorType_CheckFailed, errors.New("check was failed, please see detail")))
```

Comment:
Is it needed to use `errors.New` to record the stack trace here?

Reply:
fixed, use the standard library `errors`